Makefile

```diff
@@ -9,7 +9,8 @@ ifndef SELECTED_CONFIGS
   # when running 'make' and 'make push'.
   SELECTED_CONFIGS = \
     configs/alpine.sh \
-    configs/ubuntu.sh
+    configs/ubuntu.sh \
+    configs/arm64alpine.sh
```

Comment:
Maybe leave only the x86_64 targets as defaults, since we can't build the arm target at the same time.